apps/autoscheduler/AutoSchedule.cpp

```diff
@@ -2947,7 +2947,9 @@ IntrusivePtr<State> optimal_schedule_pass(FunctionDAG &dag,
         std::unordered_map<uint64_t, int> hashes;
         q.swap(pending);
 
-        internal_assert(!pending.empty());
```

Comment:
internal_assert should already throw an exception when WITH_EXCEPTIONS is on. Perhaps it needs to be #defined separately because this is built separately to Halide?

Reply:
Indeed, closing the PR.